src/pages/offers/offer-review.tsx

```diff
@@ -0,0 +1,28 @@
+import { Review } from '../../types';
+
+export default function OfferReview ({avatar, name, date, text}: Review): JSX.Element {
```

Comment:
И так везде. У тебя тип определяется тем, что в `return` есть

```suggestion
export default function OfferReview ({avatar, name, date, text}: Review) {
```